react/react1/week2/homework/my-app/src/components/AddTodoItem.js

```diff
@@ -0,0 +1,17 @@
+import React from "react";
+
+export const AddTodoItem = ({ addTodos, items }) => {
+  const newId = items.length;

```

Comment:
Using the array's length as a new id can lead to some items having the same id. Consider the following case:
1. The array has 3 items.
2. You add a new 4th item so it gets the id 4.
3. You remove the 1st item, so the array has 3 items again.
4. You add a new 4th, so it also gets the id 4.

One way to solve this is to use 
```Math.floor(Math.random() * 10000)``` to generate a random number.

Reply:
Thanks that  is nice 👍 , what about  Math.floor(Math.random() * 10000) can it be unique.